torch/_dynamo/variables/builder.py

```diff
@@ -878,6 +878,8 @@ def _clone_input(value):
         specialized_props["specialized_value"] = specialized_value
 
         options.update(specialized_props)
+        assert "tensor_dict" not in proxy.node.meta
+        proxy.node.meta["tensor_dict"] = example_value.__dict__
```

Comment:
You probably want to do a shallow copy

Reply:
I wasn't sure, I was actually thinking maybe a deep copy, in case this graph outlives the tensors, but I wasn't sure...

Reply:
The main reason to shallow copy is so that modifying the dict doesnt spooky action at a distance between meta and the og tensor

Reply:
sg - let's decide if we are keeping dict below, and if we are, I can rev this to add a copy